### PR TITLE
#946 : Create distinct tutorials for CLI with/without serializers (SR/Avro)

### LINKED ISSUE #946: CLIs: split into SR and non-SR
https://kafka-tutorials.confluent.io/kafka-console-consumer-producer-basics/ is a popular KT, however the 2 flavors of it are mixing up 2 concepts:

1. https://kafka-tutorials.confluent.io/kafka-console-consumer-producer-basics/confluent.html -- uses Avro
2. https://kafka-tutorials.confluent.io/kafka-console-consumer-producer-basics/kafka.html -- does not use Avro

This GH issue is to separate these out and add 2 flavors

1. https://kafka-tutorials.confluent.io/kafka-console-consumer-producer-basics/confluent.html -- create net-new
2. https://kafka-tutorials.confluent.io/kafka-console-consumer-producer-basics/kafka.html -- leave as-is
3. https://kafka-tutorials.confluent.io/kafka-console-consumer-producer-avro/confluent.html -- port from existing `kafka-console-consumer-producer-basics/confluent.html`
4. https://kafka-tutorials.confluent.io/kafka-console-consumer-producer-avro/kafka.html -- create net-new

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

#946 + pruning unused references to another tutorial

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-946
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-946/kafka-console-consumer-producer-avro/confluent.html
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-946/kafka-console-consumer-producer-avro/kafka.html
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-946/kafka-console-consumer-producer-basics/confluent.html
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-946/kafka-console-consumer-producer-basics/kafka.html

### CODE REVIEW

File: _includes/tutorials/console-consumer-producer-avro/confluent/markup/dev/answer-short.adoc
Comment (bbejeck) [inline _includes/tutorials/console-consumer-producer-avro/confluent/markup/dev/answer-short.adoc:6]: One minor suggestion - the `console-producer-keys.sh` file has the location of schema file in the command so maybe add a quick note in the short answer that the path for the schema needs to change.  But I'll leave it up to you.

**Review by bbejeck — APPROVED** [review]
Nice work @ybyzek. I have one minor comment, but otherwise this LGTM.

I verified by running the local versions of both avro and basic tutorials.

File: _includes/tutorials/console-consumer-producer-avro/confluent/markup/dev/answer-short.adoc
Comment (ybyzek) [reply _includes/tutorials/console-consumer-producer-avro/confluent/markup/dev/answer-short.adoc:7]: @bbejeck done in latest commit

**Pull request merged**
